### PR TITLE
ENH: Add University of Frankfurt

### PR DESCRIPTION
**Pull request by dragonyanglong**
This is for issue #52 .
No schools found for physics and chemistry depts. Instead, found `Institute of Theoretical Physics` under physics dept.

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -262,4 +262,12 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
+ufrankfurt:
+  aka: [University of Frankfurt]
+  city: Frankfurt
+  country: Germany
+  departments: {chemistry: {name: Department of Chemistry}, physics: {name: Department of Physics}}
```
Comment (sbillinge) [inline db/institutions.yml]: if you didn't find a department of physics, please don't add it.  Add Inst of theoretical physics, or whatever.

**Comment by sbanerjee23** [comment]
@dragonyanglong little tough to find these when you google search in English but all these big schools have these departments for sure, so you might want to use google translate and look around just a bit. For german schools you can usually just use "Physik" for Physics and "Chemie" for Chemistry. 

Physics Department: https://www.uni-frankfurt.de/60506908/Physik
Chemistry departments are usually split up into institutes in Germany like inorganic and organic, for uni frankfurt there's four of them that make up the chemistry "department": https://www.uni-frankfurt.de/52883776/Chemie

**Comment by dragonyanglong** [comment]
Thanks @sbillinge and @sbanerjee23 for the clarification and help. I did find physics and chemistry departments, so that's why I add them into the db. Actually my point here is just that there is no "school" available for U of Frankfurt, so I didn't add school into the db. Since school should be larger than dept, but in U of Frankfurt, only institutes are found, which is smaller than dept.

**Comment by sbillinge** [comment]
For the German uni's I would put the departments into the "schools" and the "institutes" into the departments.

File: db/institutions.yml [hunk 2]
```diff
@@ -262,4 +262,12 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
+ufrankfurt:
+  aka: [University of Frankfurt]
```
Comment (sbillinge) [inline db/institutions.yml]: maybe add Frankfurt University to aka

**Comment by dragonyanglong** [comment]
Thanks. I edited. Please review and I will do rebase afterwards.

**Comment by sbillinge** [comment]
I reviewed.  It looks good.  You may as well rebase too, so if it passes
review I have a better chance of being able to pull it in.

S

**Comment by dragonyanglong** [comment]
Thanks @sbillinge ! Just rebased. 
Closes #52

**Comment by sbillinge** [comment]
thanks @dragonyanglong !

**Pull request merged**